A3A/addons/core/functions/CREATE/fn_createAIOutposts.sqf

```diff
@@ -246,7 +246,8 @@ if (_spawnParameter isEqualType []) then {
 		} else {
 			(_faction get "vehiclesMilitiaTrucks") +
 			(_faction get "vehiclesMilitiaLightArmed") +
-			(_faction get "vehiclesMilitiaCars")
+			(_faction get "vehiclesMilitiaCars")+
+			(_faction get "vehiclesBasic") //we should use them somewhere at least
```

Comment:
basic shouldn't be used anywhere outside of where it already is used, for reasons i've already mentioned

Reply:
That's the point, they aren't used anywhere

Reply:
That's my point, there's a reason they aren't used anywhere

I don't see how people would like going to an outpost just to find a quadbike, or another vehicle of questionable quality

Reply:
And here is another point, you aren't getting vehicles that you want all the time on mil bases or air bases, right? Sometimes you get shitty vehicles. + I believe basics will spawn only on low war levels alongside shitty militia vehicles.

Reply:
I don't see anything in that function to only spawn basics at low war levels

Everything else looks fine, I just don't see any reason to start using basics now